### ISSUE TITLE
[Intro to iOS] Assignment 2 - Yelp #2

### ISSUE THREAD

**Issue by matrixstone** [matrixstone/Yelp#1]
Hi Timothy.
Sorry for my delayed submission. I was trying to implement the basic view and filter function but encountered with some bugs and that tool a lot of time.

Actually I still had one in the current project. Any time when I switched back to the main view from the filter view, the cells in the main view will be compressed. You can tell it from the video I attached in the README. Feels like there are some problem with the autolayout I setted up. 

Do you have a clue why it happened?

Thanks!
/cc @codepathreview

**Comment by timothy1ee** [comment]
Yeah, I mentioned it on Facebook, it's a known issue with UITableViewAutomaticDimension.

**Comment by matrixstone** [comment]
Got it!
Just fixed that issue.
Thanks!
/cc @codepathreview
